_posts/en/newsletters/2025-06-06-newsletter.md

```diff
@@ -0,0 +1,159 @@
+  header merkle root to the coinbase transaction to the witness data.
+  This ensures the data was available and uncorrupted at the time the
+  node was initially synced.  If nobody routinely validates the
+  existence of the data, it could conceivably be lost, as [has
+  happened][ripple loss] to at least one altcoin.
+
+  The discussion was ongoing at the time of writing.
+
+## Changing consensus
+
+_A monthly section summarizing proposals and discussion about changing
+Bitcoin's consensus rules._
+
+- **Quantum computing report:** Clara Shikhelman [posted][shikelman
+  quantum] to Delving Bitcoin the summary of a [report][sm report] she
+  co-authored with Anthony Milton about the risks to Bitcoin user of
+  fast quantum computers, an overview of several pathways to [quantum
+  resistance][topic quantum resistance], and an analysis of tradeoffs
+  involved in upgrading the Bitcoin protocol.  The authors find 4 to 10
+  million BTC are potentially vulnerable to quantum theft, some
+  mitigation now is possible, Bitcoin mining is unlikely to be
+  threatened by quantum computing in the short or medium term, and
+  upgrading requires widespread agreement.
+
+- **Transaction weight limit with exception to prevent confiscation:**
+  Vojtěch Strnad [posted][strnad limit] to Delving Bitcoin to propose
+  the idea for a consensus change to limit the maximum weight of most
+  transactions in a block.  The simple rule would only allow one
+  transaction per block (besides the coinbase transaction) to exceed
+  400,000 weight units (100,000 vbytes).  Strnad and others described
+  the motivation for limiting the maximum transaction weight:
+
+  - _Easier block template optimization:_ it's easier to find a
+    near-optimal solution to the [knapsack problem][] the smaller the
+    items are compared to the size of the overall limit.  This is partly
+    due to minimizing the amount of space leftover at the end, with
+    smaller items leaving less space leftover.
+
+  - _Easier relay policy:_ the policy for relaying unconfirmed
+    transactions between nodes tries to guess what transactions will be
+    mined in order to avoid wasting bandwidth.  Giant transactions make
+    guessing harder as even a small change in the top feerate can cause
+    them to be delayed or evicted.
+
+  Gregory Sanders [noted][sanders limit] it might be reasonable to
+  simply soft fork a maximum weight limit without any exceptions based
+  on Bitcoin Core's 12 years of consistent relay policy.  Gregory
+  Maxwell [added][maxwell limit] that transactions spending only UTXOs
+  created before the soft fork could be allowed an exception to prevent
+  confiscation, and that a [transitory soft fork][topic transitory soft
+  forks] could be used to allow the restriction to expire if the
+  community decided not to renew it.
+
+  Additional discussion examined the needs of the parties who might want
+  large transactions, mainly [BitVM][topic acc] users in the near-term,
```

Comment:
```suggestion
  large transactions, mainly [BitVM][topic acc] users in the near term,
```
(Noun versus adjective; I think "near-term" would be correct in a sentence like, "Near-term considerations lead us to conclude...")